### DISCUSSION TITLE
gestion des salaires

### DISCUSSION THREAD

**Discussion by douaakriba** [community/community#116716] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

je veux construire un siteweb pour La gestion du salaire du personnel Administratif et Technique au sein d’une faculté est une opération  
lourde et délicate quand elle réalisée avec le tableau Excel. Le système développé doit gérer  
l’ensemble du processus et d’étape pour aboutir à un traitement en partant du service fait de chaque  
employé.

**Comment by Akash1134** [comment]
Hi @douaakriba , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
